src/System.Net.Security/src/System/Net/Security/SslStreamInternal.cs

```diff
@@ -31,14 +31,12 @@ internal class SslStreamInternal
         private int _nestedRead;
         private AsyncProtocolRequest _readProtocolRequest; // cached, reusable AsyncProtocolRequest used for read operations
         private AsyncProtocolRequest _writeProtocolRequest; // cached, reusable AsyncProtocolRequest used for write operations
+        private Action<Task, object> _freeBufferAction;
```

Comment:
Yeah, I thought about that, I am happy to remove the logging and make it static :) But I thought it might be a step too far.